### PR TITLE
fix: styleguide should not give errors in console

### PR DESCRIPTION
**Pull request by mswertz**
How to test:
- go to https://preview-pr-emx2-854.dev.molgenis.org
- sign in (or make pet store schema public)
- go to styleguide
- open developer console and see no errors (i.e. red marked console entries)

### CODE REVIEW

File: apps/dev-proxy.config.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
-const HOST = process.env.MOLGENIS_APPS_HOST || "https://emx2.dev.molgenis.org";
```
Comment (connoratrug) [inline apps/dev-proxy.config.js:1]: why does this give console errors ? , suggest to keep default pointing to dev server

File: apps/dev-proxy.config.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+const HOST = process.env.MOLGENIS_APPS_HOST || "http://localhost:8080";
 const SCHEMA = process.env.MOLGENIS_APPS_SCHEMA || "pet%20store";
 
-const opts = { changeOrigin: true, secure: false }
+const opts = { changeOrigin: true, secure: false };
 
 module.exports = {
-  "^/graphql": {
```
Comment (connoratrug) [inline apps/dev-proxy.config.js:7]: what issue does this solve , how can i test that this change fixes the issue ( if any)

File: apps/styleguide/src/display/LinksList.vue [hunk 3]
```diff
@@ -32,12 +32,19 @@ export default {
...
-    { label: 'Vue',  href: 'https://vuejs.org/' } 
-]
+<script>
+  export default {
+    data() {
+      return {
+        links: [{label: 'Molgenis', href: 'https://molgenis.org'}]
+      }
+    }
+  }
```
Comment (connoratrug) [inline apps/styleguide/src/display/LinksList.vue:42]: this makes me a bit sad , is all the js vue needed to define a example ?, what issue is being fixed by this change ?

File: apps/styleguide/src/display/LinksList.vue [hunk 4]
```diff
@@ -32,12 +32,19 @@ export default {
...
+</script>
 <template>
-  <links-list :items="links" ><p></p>Sample content</p></<links-list>
+  <LinksList :items="links">
```
Comment (connoratrug) [inline apps/styleguide/src/display/LinksList.vue:45]: lower case no is the html tag default / suggest style , suggest to keep it

File: apps/styleguide/src/mixins/TableMetadataMixin.vue [hunk 5]
```diff
@@ -1,3 +1,6 @@
+<template>
+  <div>Not meant for direct use</div>
```
Comment (connoratrug) [inline apps/styleguide/src/mixins/TableMetadataMixin.vue:2]: suggest to move it outside of example folder

File: apps/styleguide/src/mixins/TableMixin.vue [hunk 6]
```diff
@@ -1,3 +1,6 @@
+<template>
+  <div>Not meant for direct use</div>
```
Comment (connoratrug) [inline apps/styleguide/src/mixins/TableMixin.vue:2]: same

File: apps/dev-proxy.config.js (hunk 1 above)
Comment (mswertz) [reply apps/dev-proxy.config.js:1]: Agreed, this should not have been in the commit

File: apps/styleguide/src/display/LinksList.vue (hunk 3 above)
Comment (mswertz) [reply apps/styleguide/src/display/LinksList.vue:42]: no, that is indeed not needed in this case. Can be example in one line.

File: apps/styleguide/src/mixins/TableMetadataMixin.vue (hunk 5 above)
Comment (mswertz) [reply apps/styleguide/src/mixins/TableMetadataMixin.vue:2]: I like to have this for testing purposes. So replaces with useful template.

File: apps/styleguide/src/mixins/TableMixin.vue (hunk 6 above)
Comment (mswertz) [reply apps/styleguide/src/mixins/TableMixin.vue:2]: ditto as above

File: apps/styleguide/src/display/LinksList.vue (hunk 4 above)
Comment (mswertz) [reply apps/styleguide/src/display/LinksList.vue:45]: hmm, I am not a fan of that. But changed it to make you happy :-)

File: apps/dev-proxy.config.js (hunk 2 above)
Comment (mswertz) [reply apps/dev-proxy.config.js:7]: When I start with proxy to localhost:8080 I get following error with ^/ and not when /
image

File: apps/styleguide/src/display/LinksList.vue [hunk 7]
```diff
@@ -32,13 +32,6 @@ export default {
...
 ```
-const links = [
-    { label: 'Molgenis', href: 'https://molgenis.org', },
-    { label: 'Vue',  href: 'https://vuejs.org/' } 
-]
-<template>
-  <links-list :items="links" ><p></p>Sample content</p></<links-list>
-</template>
-
+<links-list :items="[{label: 'Molgenis', href: 'https://molgenis.org'}]"/>
```
Comment (connoratrug) [inline apps/styleguide/src/display/LinksList.vue:35]: single item list is not a great example of a list rendering component

**Review by connoratrug — APPROVED** [review]

File: apps/styleguide/src/display/LinksList.vue (hunk 7 above)
Comment (mswertz) [reply apps/styleguide/src/display/LinksList.vue:35]: indeed, will add another item before merging :-)

**Pull request merged**